src/python/test/test_dxclient.py

```diff
@@ -2811,6 +2812,60 @@ def _get_analysis_id(dx_run_output):
         # don't actually need it to run
         run("dx terminate " + watched_job_id)
 
+        # warning when --priority is normal/low with --watch
+        try:
+            dx_run_output = run("dx run myapplet -y --watch --priority normal --brief")
+        except subprocess.CalledProcessError:
+            # ignore any watch errors; just want to test requested
+            # priority
+            pass
+        expected_warning = dx_run_output.split('\n\n')[0]
+        watched_job_id = dx_run_output.split('\n\n')[1].strip()
+        watched_job_desc = dxpy.describe(watched_job_id)
+        self.assertEqual(watched_job_desc['applet'], applet_id)
+        self.assertEqual(watched_job_desc['priority'], 'normal')
+        for string in ["WARNING", "normal", "interrupting interactive work"]:
+            self.assertIn(string, expected_warning)
+
+        # don't actually need it to run
+        run("dx terminate " + watched_job_id)
+
+        # warning when --priority is normal/low with --ssh
+        try:
+            dx_run_output = run("dx run myapplet -y --ssh --priority low --brief")
+        except subprocess.CalledProcessError:
+            # ignore any ssh errors; just want to test requested
+            # priority
+            pass
+        expected_warning = dx_run_output.split('\n\n')[0]
+        ssh_job_id = dx_run_output.split('\n\n')[1].strip()
+        ssh_job_desc = dxpy.describe(ssh_job_id)
+        self.assertEqual(ssh_job_desc['applet'], applet_id)
+        self.assertEqual(ssh_job_desc['priority'], 'low')
+        for string in ["WARNING", "low", "interrupting interactive work"]:
+            self.assertIn(string, expected_warning)
+
+        # don't actually need it to run
+        run("dx terminate " + ssh_job_id)
```

Comment:
Both ssh tests would not work due to the error below and failure to connect to aws server.

> Pseudo-terminal will not be allocated because stdin is not a terminal.

I guess the reason why the test in L2785 worked is that the error is passed and the job id is still the `watched_job_id`. @commandlinegirl